src/email_requests/views.py

```diff
@@ -51,9 +52,9 @@ def post(self, request, *args, **kwargs):
         new_request = Request(
             mentee=mentee_profile,
             mentor=mentor,
-            email_body=content_string,
+            email_body=user_message,
```

Comment:
why not content_string?

Reply:
Because the rest of the email string will be the same for everyone, no use storing duplicates

Reply:
but if we don't store the duplicates, the email body we display to users later will have to be generated again. disk is actually fairly cheap. we don't need premature optimizations. unless, we need just the user body part, in which case we can do this

Reply:
Ram gave me the sense that when the frontend is listing requests, they only want to show the user message, not the entire email

Reply:
Yeah so we decided that on the front end, in the requests tab, we display all requests related to the current user (becuase of the fact that they can toggle btwn being an ambassador and mentee. If they made the request we would display: "You requested blank:" with the message body and if they received a request we would display "You received a request from blank" with the message as well. Timeline style

Reply:
So it would be useful if when we get /requests/list/me/ we get ALL requests (sent or received) from the current user, with just the message body so we can just display that instead of fishing it out from the email body